ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml

```diff
@@ -3448,6 +3448,17 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-vsphere-ipi-ovn-custom-transit-switch-masquerade-subnet
+- as: vsphere-ipi-ovn-winc-f7
+  cron: 35 3 5 * *
```

Comment:
Test name show as `f7` (weekyly), while cron entry is monthly

Reply:
Thanks, updated to `3 18 7,16,23,30 * *`